controllers/mars/mars.go

```diff
@@ -74,9 +74,9 @@ func marsConfigInJson(marJob *v1alpha1.MarsJob, rtype, index string) (string, er
 
 	// Mars worker will perceive allocated resources and real-time usage inside worker
 	// process and report to scheduler.
-	if v1.ReplicaType(rtype) == v1alpha1.MarsReplicaTypeWorker {
+	if rtype == strings.ToLower(string(v1alpha1.MarsReplicaTypeWorker)) {
```

Comment:
is it better to lowercase rtype as well ?

Reply:
done